v1.3 RC omnibus (#2104)
fishtown-analytics/docs.getdbt.com: website/docs/reference/node-selection/methods.md

</VersionBlock>

<VersionBlock firstVersion="1.3">

> While most config values are strings, the `config` method can also be used to match boolean configs, dictionary keys, and values in lists.

Reviewer: ```suggestion
While most config values are strings, you can also use the `config` method to match boolean configs, dictionary keys, and values in lists.
```

For example, given a model with the following configurations:
```
{{ config(
  materialized = 'incremental',
  unique_key = ['column_a', 'column_b'],
  grants = {'select': ['reporter', 'analysts']},
  transient = true
) }}

select ...
```

> It can be selected using any of the following:

Reviewer: switch to active
```suggestion
 You can select using any of the following:
```
